sessionctx/variable/sysvar.go

```diff
@@ -685,6 +685,30 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBEnablePreparedPlanCache, Value: BoolToOnOff(DefTiDBEnablePreparedPlanCache), Type: TypeBool, SetGlobal: func(s *SessionVars, val string) error {
+		EnablePreparedPlanCache.Store(TiDBOptOn(val))
+		return nil
+	}, GetGlobal: func(s *SessionVars) (string, error) {
+		return BoolToOnOff(EnablePreparedPlanCache.Load()), nil
+	}},
+	{Scope: ScopeGlobal, Name: TiDBPreparedPlanCacheSize, Value: strconv.FormatUint(uint64(DefTiDBPreparedPlanCacheSize), 10), Type: TypeUnsigned, MinValue: 1, MaxValue: math.MaxUint32, SetGlobal: func(s *SessionVars, val string) error {
```

Comment:
For the MaxValue, lets check with @yudongusa if we can use something lower than `math.MaxUint32` to reduce the risk of misconfiguration. I assume the unit is "items" (not memory), so I would propose 100K might be reasonable?

Reply:
The current plan-cache is session-level, 100K looks good to me. @morgo